tidb-cloud/integrate-tidbcloud-with-netlify.md

```diff
@@ -0,0 +1,59 @@
+
+This guide describes how to connect your TiDB Cloud clusters to Netlify projects.
+
+## Prerequisites
+
+Before connecting, make sure the following prerequisites are met.
+
+### A Netlify account and a deployed site
+
+You are expected to have an account and a site in Netlify. If you do not have any, refer to the following links to create one:
+
+* [Sign up a new account](https://app.netlify.com/signup).
+* [Add a site](https://docs.netlify.com/welcome/add-new-site/) in Netlify. If you do not have an application to deploy, you can use the [TiDB Cloud Starter Template](https://github.com/tidbcloud/nextjs-prisma-example#deploy-on-netlify) to have a try.
+
+### A TiDB Cloud account and a TiDB cluster
+
+You are expected to have an account and a cluster in TiDB Cloud. If you do not have any, refer to [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md).
+
+One TiDB Cloud cluster can connect to multiple Netlify sites.
+
+### All IP addresses allowed for traffic filter in TiDB Cloud
+
+For Dedicated Tier clusters, make sure that the traffic filter of the cluster allows all IP addresses (set to `0.0.0.0/0`) for connection, this is because Netlify deployments use dynamic IP addresses.
+
+Serverless Tier clusters allow all IP addresses for connection by default, so you do not need to configure any traffic filter.
+
+## Connect via manually setting environment variables
+
+To use this method, make sure that your TiDB Cloud cluster [allows all IP addresses access](#all-ip-addresses-allowed-for-traffic-filter-in-tidb-cloud) and save your root password in the [Security Settings](/tidb-cloud/configure-security-settings.md) dialog.
+
+1. Follow the steps in [Connect to a TiDB Cloud cluster via standard connection](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) to get the connection information of your TiDB cluster.
+2. Go to your **Netlify dashboard** > **Netlify project** > **Site settings** > **Environment Variables**, and then [update variables](https://docs.netlify.com/environment-variables/get-started/#update-variables-with-the-netlify-ui) according to the connection information of your TiDB cluster.
+
+    - **Key**: `DATABASE_URL` 
+    - **Values**: `mysql://<User>:<Password>@<Endpoint>:<Port>/<Database>?sslaccept=strict`
+
+The following is a datasource example in the Prisma schema file for a TiDB Cloud Serverless Tier cluster:
+
+```
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
+}
+```
+
+In Netlify, you can declare the environment variables as follows. You can customize the name according to your project need.
+
+- **Key** = DATABASE_URL 
+- **Values** = `mysql://<User>:<Password>@<Endpoint>:<Port>/<Database>?sslaccept=strict`
```

Comment:
1. Since this description repeats step 2, can you remove it?
2. "You can customize the name" means you can customize the **Key**?